app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def new
+    @comments = Comment.new
+  end
+
+  def create
+    new_comment = Comment.new(text: params[:comment][:text], post: Post.find(params[:post_id]),
+                              author: current_user)
+    if new_comment.save
+      redirect_to user_posts_path
+    else
+      render :new
+    end
+  end
```

Comment:
### [OPTIONAL]
Please follow the instructions mentioned in the Posts Controller. 👍